modules/apps/commerce/commerce-product-content-web/src/main/resources/META-INF/resources/product_detail/render/js/Tabs.js

```diff
@@ -0,0 +1,145 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import ClayTabs from '@clayui/tabs';
+import React, {useEffect, useState} from 'react';
+
+let tabs = [];
+
+function display(tabId) {
+	for (let i = 0; i < tabs.length; i++) {
+		const currentTabId = tabs[i].id;
+
+		if (tabId === currentTabId) {
+			document.getElementById(tabId).classList.add('active', 'show');
+		}
+		else {
+			document
+				.getElementById(currentTabId)
+				.classList.remove('active', 'show');
+		}
+	}
+}
+
+export default function ({
+	hasCPDefinitionSpecificationOptionValues,
+	hasCPMedia,
+	hasDescription,
+	hasReplacements,
+	navCPMediaId,
+	navDescriptionId,
+	navReplacementsId,
+	navSpecificationsId,
+	portletNamespace,
+}) {
+	const [activeTabKeyValue, setActiveTabKeyValue] = useState(0);
+
+	useEffect(() => {
+		tabs = document.getElementsByClassName(
```

Comment:
Why did you move this here?

Reply:
because otherwise I can't use the namespace outside of the function, I need to load the tabs when the component is loaded, one way is to use useEffect

Reply:
Right, is this affecting the tabs loading?

Reply:
the tabs are not loading if you puts that getElementByClassName outside of the react function. But with this it works smoothly with no problems

Reply:
Yeah, I got that already. I was asking if there is some glitch during the react component loading